### PR TITLE
change kubepath to take the raw bytes of the file

### PR DESCRIPTION
**Pull request by obadrawi**
Change the kube_path provider property to kube_file which takes the raw text of the kubeConfig file. You can find how to do that in the examples folders

### CODE REVIEW

File: examples/aws/main.tf [hunk 1]
```diff
@@ -1,7 +1,29 @@
 provider "gardener" {
   profile            = "<my-gardener-project>"
   aws_secret_binding = "<my-gardener-aws-secret>"
-  kube_path          = "<my-gardener-service-account-kubeconfig>"
+  kube_path          = "${file("<my-gardener-service-account-kubeconfig>")}"
```
Comment (clebs) [inline examples/aws/main.tf]: Shouldn't this property be called `kube_file`? Same for the other .tf files

File: provider/provider.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ func Provider() terraform.ResourceProvider {
 				Required:    true,
 				DefaultFunc: schema.EnvDefaultFunc("PROFILE", ""),
 			},
-			"kube_path": {
+			"kube_file": {
 				Type:        schema.TypeString,
 				Required:    true,
 				DefaultFunc: schema.EnvDefaultFunc("KUBECONFIG", ""),
```
Comment (clebs) [inline provider/provider.go]: This default value is no longer valid.

File: examples/azure/main.tf [hunk 3]
```diff
@@ -1,7 +1,29 @@
...
...0NBUW9DZ2dFQkFOWEkvZ0E3YWVpVkxLVGVCVEpxYk5hM3JrOHpaVUcrNXlnLy9WU1RlMlVtb21rZworMlNxY291Zis4S05FM0ZSL1hyZTR5cWM1S2dXWEdKUWdxMWl1a3RWdnVBRzFSRjZjamFMQlE0MWNZNHF6TGJ6Ci9TdFhIRVVMR3NhRENGaGRKZ0wzNXJjbHh0Zi9OcGRvczczUzhvQnV2djFPVk1NSDY3Y0toRkxOajFSZWdxcHMKN2ZhcXAyZXJsNVB1UlZiSjY5d29xK0E3YS9GUktKN3JBbUNrUzhaeHRaVHZ3Y2ZVUDNPbzl4bEdlQkh1VUlvUgowMW9HYlc2WGpjNDNTbytlcFRkUXBxZTNWNjhqeGgybkV4NXFheDZ4YjJJTC9VUXlpQlRpbEhFTkN2Tk1WZ2JqCnE4TUNCSjduUGhoZGkvRTJrZjRYY2FGRXZycHN0YzNQeTFXV1NVRUNBd0VBQWFObU1HUXdEZ1lEVlIwUEFRSC8KQkFRREFnRUdNQklHQTFVZEV3RUIvd1FJTUFZQkFmOENBUUl3SFFZRFZSME9CQllFRkxubkNJWVlwdHdhQ05SbwpXbGgrOWgwd2J0WlhNQjhHQTFVZEl3UVlNQmFBRkxubkNJWVlwdHdhQ05Sb1dsaCs5aDB3YnRaWE1BMEdDU3FHClNJYjNEUUVCQ3dVQUE0SUJBUUJZODhSWWFZb0NBVXlxSmk4VDFyeE5ENEJQeUYxZTlFemVBMGhXdUVjbzdFQmcKazdDU2NlamFyNVRUNDM2L1kvV1Mra0Z3bUF1L3I3OWtIMTB3MVA3UnI2dEFCZ1BJci91MExVako5NXc1eG5idQphTW9DdVB4SXY2SUdCNzVhZkhUVkxRbkNGRDBZS0NmRXVua2JZS09veWJWRGJwYUwyNDhWSnl6cURac0ZRU2V5ClJ1QkhoeTdtRzRHSlozaktwdExVU3JOMmowemV2U2E2Vmo1L2Vic2g0UDgvVXBteEpuMXRoRGZJNUcrdEZ0cTEKbmdVTkRsTjJWejNJdThBLzdSVlRxb1hGTzREcGJaUHFnR2FyZUVJWmNPN1JzYjBKc0xFZzFwWGpvVmo0bkNYSgphMmdadWdBVnFZOMVBROFp0bFRKbDFnUSt2bUljTlRtakFWCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0=
```
Comment (clebs) [inline examples/azure/main.tf]: Please delete any certificates or tokens, this could pose a security issue.

File: examples/aws/main.tf (hunk 1 above)
Comment (obadrawi) [reply examples/aws/main.tf]: fixed it

File: provider/provider.go (hunk 2 above)
Comment (obadrawi) [reply provider/provider.go]: removed it

File: examples/azure/main.tf (hunk 3 above)
Comment (obadrawi) [reply examples/azure/main.tf]: fixed it

File: examples/README.md [hunk 4]
```diff
@@ -7,13 +7,39 @@ This folder contains a set of examples which use Gardener services to deploy [aw
 provider "gardener" {
   profile            = "<my-gardener-project>"
   <provider>_secret_binding = "<my-gardener-<provider>-secret>"
-  kube_path          = "<my-gardener-service-account-kubeconfig>"
+  kube_file          = "${file("<my-gardener-service-account-kubeconfig>")}"
 }
 ```
+You can pass the kube_file using the raw text alone as follows:
+```bash
+kube_path          =<<-EOT
```
Comment (clebs) [inline examples/README.md]: This still has the old property name.

Comment (obadrawi) [reply examples/README.md]: changed it

**Review by clebs — APPROVED** [review]

**Review by bszwarc — APPROVED** [review]

**Pull request merged**